BizHawk.Client.EmuHawk/tools/Debugger/GenericDebugger.Disassembler.cs

```diff
@@ -75,7 +77,7 @@ private void DisassemblerView_QueryItemText(int index, int column, out string te
 			{
 				if (column == 0)
 				{
-					text = string.Format($"{{0:X{_pcRegisterSize}}}", _disassemblyLines[index].Address);
+					text = _disassemblyLines[index].Address.ToHexString(_pcRegisterSize);

```

Comment:
Why are you removing ToHexString() in some places, and adding it in others?

Reply:
I'm removing it "where param numdigits is constant" (in the commit message). In the two places I switch to the helper method, the width is only known at runtime (as far as I could tell).